src/Init/Data/Char/Order.lean

```diff
@@ -0,0 +1,25 @@
+/-
+Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Reichert
+-/
+module
+
+prelude
+public import Init.Data.Char.Basic
+import Init.Data.Char.Lemmas
+public import Init.Data.Order.Factories
+
+open Std
+
+public instance : OrderData Char := .ofLE Char
+
+public instance instIsLinearOrder : IsLinearOrder Char := by
```

Comment:
Are you missing a namespace here? It seems like a rather general instance name to me.